qemu/tests/x86_cpu_model.py

```diff
@@ -72,6 +72,16 @@ def run(test, params, env):
     cmd = params["get_model_cmd"]
     out = session.cmd_output(cmd)
     if not re.search(guest_model, out):
+        check_cmd = params.get("check_cmd")
+        if check_cmd:
+            vulnerabilities = params.get("vulnerabilities").split()
+            for item in vulnerabilities:
```

Comment:
1. Suggest to compare vulnerability one by one. We may have new vulnerability in the future, not sure if they will be same in host and guest. 
2. Only compare the keywords, like 'Not afftected', 'Mitigation'. The sentence after that might differ from guest to host.

Reply:
> 1. Suggest to compare vulnerability one by one. We may have new vulnerability in the future, not sure if they will be same in host and guest.
> 2. Only compare the keywords, like 'Not afftected', 'Mitigation'. The sentence after that might differ from guest to host.

Thanks and update, please help to review.